sonar-jmeter-plugin/src/main/java/org/sonar/plugins/jmeter/JMeterPostJob.java

```diff
@@ -85,8 +89,8 @@ public void executeOn(Project project, SensorContext context) {
   /**
    * Gets the generated jtl file path, if it was any generated
    */
-  protected String getJtlFilePath(Project project, String innerProjectJMeterReportsPath) {
-    String baseDirPath = project.getFileSystem().getBasedir().getAbsolutePath();
+  protected String getJtlFilePath(String innerProjectJMeterReportsPath) {
+    String baseDirPath = filesystem.baseDir().getAbsolutePath();
```

Comment:
This is only available starting from 5.0 but you could replace this by filesystem.resovePath(innerProjectJMeterReportsPath)